contracts/utils/resolvers.js

```diff
@@ -16,7 +17,7 @@ const resolveAsset = async (symbol) => {
 
   // Not using helpers here as they import hardhat which won't work for Hardhat tasks
   if (process.env.FORK === "true" || hre.network.name != "hardhat") {
-    const { chainId } = await hre.ethers.provider.getNetwork();
+    const { chainId } = await ethers.getDefaultProvider().getNetwork();
```

Comment:
It turns out this will only work for mainnet :-(

https://github.com/NomicFoundation/hardhat/issues/2424#issuecomment-1066816392 

I'll see if I can get something that will work with Holesky that doesn't use Hardhat

Reply:
I've moved `registerValidators` and `stakeValidators` into `utils` without a Hardhat dependency